.github/actions/silences-report-expired/silences-report-expired.sh

```diff
@@ -125,13 +148,14 @@ report() {
       ;;
     *)
       pr_link=$(_run gh pr create \
-               --head "$branch_name" \
-               --reviewer "${userGithubHandle}" \
-               --assignee "${userGithubHandle}" \
-               --title "${message}" \
-               --body "$PULL_REQUEST_BODY")
-      # Ignore auto merge error, which might be due to repository settings
-      _run gh pr merge --squash --auto "$branch_name" || true
+              --head "$branch_name" \
+              --reviewer "${userGithubHandle}" \
+              --assignee "${userGithubHandle}" \
+              --title "${message}" \
+              --body "$PULL_REQUEST_BODY")
```

Comment:
The PULL_REQUEST_BODY message should be updated to indicate whether the silence has expired already or is about to expire in 2 days.